bats/fb-content-katello.bats

```diff
@@ -161,8 +161,11 @@ EOF
   tPackageInstall walrus-0.71 && tPackageExists walrus-0.71
 }
 
+@test "wait for errata to become available" {
+  sleep 60
```

Comment:
This should probably just be a "run" since its not really testing anything.

Reply:
On the other hand, when does sleep fail other than being interrupted?